docs/run.rst

```diff
@@ -48,10 +48,7 @@ Options
     Public node base URL. Default is ``https://mainnet-tezos.giganode.io``. This argument will only be used in case the reward provider is set to ``prpc``. This node will only be used to query reward data and delegator list. It must be an ARCHIVE node.
 
 ``-r --reports_base <path>``
-    Directory to create reports. Default value: ``~/pymnt/reports``.
-
-``-f --config_dir <path>``
-    Directory to find baking configuration. Default value: ``~/pymnt/cfg``.
+    Directory for reports and configurations. Default value: ``../trd-data``.
```

Comment:
I am not sure why ../trd-data ?

Reply:
probably better to go with ~/trd-data, and expand user when using it.

Reply:
OK, maybe the MAIN problem was the simulations always being in one place and the easiest solution is the current solution of my latest commit? How to expand ~ properly on windows btw?

Reply:
You use expand_user, for reference: https://www.geeksforgeeks.org/python-os-path-expanduser-method/
It's also in email.py iirc if you want an example.

Reply:
I think we are using that function everywhere already. I read that it should work on Windows as well.